### PR TITLE
Added the Welcome animation

### PR DESCRIPTION
**Pull request by KC1064**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [Done] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [Done] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [Done] Your pull request contains `index.html` and `styles.css`.
	- [Done] Your html file is named exactly: `index.html`
	- [Done] Your css file is named exactly: `styles.css`
- [Done] Your pull request your `meta.json` with:
	- [Done] `githubHandle`: being your unique github user name,
	- [Done] `artName` : name of your animation
- [Done] Your submission includes **at least one animation**.
- [Done] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [Done] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/KC1064-Welcome/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome-Animation</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (LaurelineP) [inline Art/KC1064-Welcome/index.html:7]: ## Stylesheet name
- [ ] your css file should be named `styles.css` 
- [ ] make sure you adjust this ling tag accordingly after changing the name

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @KC1064 🙌
Thanks for your contribution.
Before we could merge your request, please make sure to address the provided feedbacks :)

File: Art/KC1064-Welcome/index.html (hunk 1 above)
Comment (KC1064) [reply Art/KC1064-Welcome/index.html:7]: Sure....I will resolve the issue and raise a PR again. Thanks for the review and feedback.

**Comment by KC1064** [comment]
Updated the Link tag

**Comment by KC1064** [comment]
Hey @LaurelineP,
Can you check again and give me an update if the issue is resolved or not?

**Pull request merged**